src/mm-grid-column/mm-grid-column.js

```diff
@@ -155,6 +134,14 @@
 		_onTrackEnd: function(e) {
 			this.set('width', this.desiredWidth + 'px');
 			this.fire(this.RESIZE_END_EVENT, { field: this.field, val: this.desiredWidth - this.startWidth });
+		},
+
+		_hasInfoTip: function(info) {
+			if (info && info.length > 0) {
```

Comment:
super pedantic but empty string is falsy so this fn could probably be simplified to `return !info`